### PR TITLE
Migration/l10n br data account service

### PR DESCRIPTION
**Pull request by renatonlima**

### CODE REVIEW

**Comment by rvalyi** [comment]
@renatonlima LGTM but please then squash commit 510b575

**Comment by renatonlima** [comment]
@rvalyi Done!

File: l10n_br_account_product/models/account_invoice.py [hunk 1]
```diff
@@ -861,8 +861,10 @@ def _validate_taxes(self, values):
         freight_value = values.get('freight_value', 0.0) or self.freight_value
         other_costs_value = values.get(
             'other_costs_value', 0.0) or self.other_costs_value
-        tax_ids = values.get('invoice_line_tax_id', [[6, 0, []]])[
-            0][2] or self.invoice_line_tax_id.ids
+        tax_ids = []
+        if values.get('invoice_line_tax_id'):
+            tax_ids = values.get('invoice_line_tax_id', [[6, 0, []]])[
+                0][2] or self.invoice_line_tax_id.ids
```
Comment (rvalyi) [inline l10n_br_account_product/models/account_invoice.py:867]: @renatonlima is it normal to touch l10n_br_account_product in this PR?
I mean is it on purpose or by mistake?

Comment (renatonlima) [inline l10n_br_account_product/models/account_invoice.py:867]: I made it to fix the tests in l10n_br_account_service

Comment (rvalyi) [inline l10n_br_account_product/models/account_invoice.py:867]: @renatonlima right, I see. Looks fine.

**Comment by rvalyi** [comment]
:+1:

**Comment by mileo** [comment]
@mstuttgart Can you review this today please?

**Comment by mileo** [comment]
:+1:

**Pull request merged**